docker/ci/install-kubectl.bash

```diff
@@ -15,6 +15,7 @@
 
 set -e
 
-wget -q https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip
-unzip -qq protoc-3.7.1-linux-x86_64.zip -d /usr/local
-rm protoc-3.7.1-linux-x86_64.zip
```

Comment:
It seems that we can use `curl -o ` to save the downloaded file to the correct place without a `mv` command.